pkg/certifier/scorecard/README.md

```diff
@@ -0,0 +1,78 @@
+# Scorecard Fetcher Integration with GUAC
```

Comment:
Instead of adding this to a README in the repo, it's probably better as a contribution to guacsec/guac-docs